README.md

```diff
@@ -59,8 +60,8 @@ JHOVE is currently being maintained by the
 
 Pre-requisites
 --------------
- 1. Java JRE 1.6  
```

Comment:
The two trailing spaces were a line brake.

Reply:
Ahhh, I suspect my editor may have had it's way here will check.